main_page.py

```diff
@@ -0,0 +1,71 @@
+from selenium.webdriver.remote.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as ec
+import my_account_page
+
+
+class Constant_newsletter:
+    newsletter_name = 'adsdada'
+    newsletter_mail = "assd@gmail.com"
+    input_name_box = (By.ID, 'es_txt_name')
+    input_mail_box = (By.ID, 'es_txt_email')
+    subscribe_button = (By.ID, 'es_txt_button')
+    confirmation_text_selector = (By.ID, 'es_msg')
+    confirmation_text = 'Successfully Subscribed'
```

Comment:
```suggestion
    newsletter_name: str = 'adsdada'
    newsletter_mail: str = "assd@gmail.com"
    input_name_box: Tuple[hgw_co_to_za_typ, str] = (By.ID, "es_txt_name")
    input_mail_box: Tuple[hgw_co_to_za_typ, str] = (By.ID, 'es_txt_email')
    subscribe_button: Tuple[hgw_co_to_za_typ, str] = (By.ID, 'es_txt_button')
    confirmation_text_selector: Tuple[hgw_co_to_za_typ, str] = (By.ID, 'es_msg')
    confirmation_text: str = 'Successfully Subscribed'
```
Fajnie by było pododawać typing do nazw. PS. zeby korzystać z Tuple musisz mieć `from typing import Tuple`